app/importers/est_data.rb

```diff
@@ -0,0 +1,62 @@
+class EstData
+  include Importer
+  LOGIN_URL = 'https://admin.export.gov/site_login'
+  ENDPOINT = 'https://admin.export.gov/admin/envirotech_solutions.json'
+
+  cattr_accessor :web_auth, :username, :password
+
+  COLUMN_HASH = {
+    'id'              => :source_id,
+    'name_chinese'    => :name_chinese,
+    'name_english'    => :name_english,
+    'name_french'     => :name_french,
+    'name_portuguese' => :name_portuguese,
+    'name_spanish'    => :name_spanish,
+    'created_at'      => :source_created_at,
+    'updated_at'      => :source_updated_at,
+  }.freeze
+
+  def initialize(resource = ENDPOINT)
+    @resource = resource
+  end
+
+  def import
+    data = JSON.parse(fetch_data)
+    articles = data.map { |article_hash| process_article_info article_hash }
+
+    Est.index articles
+  end
+
+  private
+
+  def fetch_data
+    headless_login
+    mechanize_agent.get(@resource).body
+  end
+
+  def headless_login
```

Comment:
Don't worry about testing this. The way that you have mocked `fetch_data` in the specs is appropriate. Since all it does is communiate over the internet with the external service holding the source docs, we don't need to provide coverage of it. We can add `# :nocov:` comments around it to tell simplecov not to complain that it hasn't been covered.